wagtail/admin/templates/wagtailadmin/home/workflow_objects_to_moderate.html

```diff
@@ -89,14 +89,5 @@
         </table>
     {% endpanel %}
 
-    <script src="{% versioned_static 'wagtailadmin/js/workflow-action.js' %}"></script>
-    <script>
-        document.querySelectorAll('[data-controller="w-dropdown"]').forEach((e) => {
-            e.addEventListener(
-                'w-dropdown:shown',
-                () => ActivateWorkflowActionsForDashboard('{{ csrf_token|escapejs }}'),
```

Comment:
We can now access the CSRF token from the JS directly via `WAGTAIL_CONFIG.CSRF_TOKEN`, so we no longer need to use inline scripts to pass the token.